api/proto/teleport/plugins/v1/plugin_service.proto

```diff
@@ -113,6 +113,18 @@ message GetAvailablePluginTypesResponse {
   repeated PluginType plugin_types = 1;
 }
 
+// SearchPluginStaticCredentialsRequest is the request type for SearchPluginStaticCredentials
+message SearchPluginStaticCredentialsRequest {
+  // Labels are matched against static credentials objects and returned.
```

Comment:
I would mention here the origin of the labels and how we expect them to match the plugin. I think that's a likely common question folks will have reading this.